### ISSUE TITLE
[Bug?] Multiple Invocations Model

### ISSUE THREAD

**Issue by googleoblivion** [darktrovx/interact#24]
Two different resources adding model interactions to the same model hash, adds overlapping points. How would you append 2nd resources point into first resources point?

Such as target behavior with overlapping model interactions.

**Comment by googleoblivion** [comment]
![image](https://github.com/darktrovx/interact/assets/2222973/0c31c10a-4dc3-4b15-b294-0ddd798f43b0)

**Comment by StrixLuca** [comment]
jup i have the same error

**Comment by Griefa** [comment]
Yup, same issue but with `AddGlobalVehicleInteraction`. ~~Also doesnt work when you have multiple options in the one export.~~ 

Ignore me, I'm *slighty* retarded

Edit, again: Turns out it doesnt work
